src/server/scripts/Spells/spell_paladin.cpp

```diff
@@ -95,11 +95,66 @@ enum PaladinSpells
     SPELL_PALADIN_ZEAL_AURA                      = 269571,
 };
 
+enum PaladinCovenantSpells
+{
+    SPELL_PALADIN_ASHEN_HALLOW                   = 316958,
```

Comment:
Unused

Reply:
I was thinking of keeping it because I wanted to implement the legendary effect related to it later today and I need to access the spell.

Reply:
It is used to access the areatrigger of the spell 😃

Reply:
Indeed, tiredness trolled